stochss/handlers/util/ensemble_simulation.py

```diff
@@ -135,8 +154,15 @@ def run(self, preview=False, verbose=True):
         if verbose:
             log.info("The ensemble simulation has completed")
             log.info("Storing the results as pickle and csv")
-        self.__store_results(results=results)
+        if not 'results' in os.listdir():
```

Comment:
This call will fail if there's a file named `results` in the same directory.

Reply:
Thats correct, this part of the run function is only executed when a job is run and we change the working directory prior to running.